mitodl/open-discussions
# Add response mail template and implement for frontpage digests

#### What are the relevant tickets?
Fixes #462 
Fixes #463 

#### What's this PR do?
Adds responsive mail template and implements it for frontpage digest emails

#### How should this be manually tested?

- Ensure your OD is configured for email sending
- Double check that you have no `EmailNotification` records
- Ensure you have posts on your user's frontpage
- Ensure your user has notification settings via `./manage.py populate_notification_settings --username USERNAME`, set them as daily for frontpage
- Run:
```python
from notifications import tasks
tasks.send_daily_frontpage_digests()
tasks.send_unsent_email_notifications()
```
- That should have triggered a batch task on your worker, ensure the logs don't contain error output and that you received the email.

#### Any background context you want to provide?

- You may need to set valid profile image urls (I suggest just giving it a public internet url to an image) as MM image urls don't work with OD locally at the moment.

#### Screenshots (if appropriate)

See https://github.com/mitodl/open-discussions/issues/450#issuecomment-363126757
